### PR TITLE
5 packages from smimram/ocaml-pandoc at 0.1.0

### PR DESCRIPTION
**Pull request by smimram**
This pull-request concerns:
-`pandoc.0.1.0`: Library to write pandoc filters
-`pandoc-abbreviations.0.1.0`: Pandoc filter to add non-breaking spaces after abbreviations
-`pandoc-crossref.0.1.0`: Pandoc filter to have LaTeX cross-references
-`pandoc-include.0.1.0`: Pandoc filter to include other files
-`pandoc-inspect.0.1.0`: Pandoc filter to inspect pandoc's JSON

---
* Homepage: https://github.com/smimram/ocaml-pandoc
* Source repo: git+https://github.com/smimram/ocaml-pandoc.git
* Bug tracker: https://github.com/smimram/ocaml-pandoc/issues

---
:camel: Pull-request generated by opam-publish v2.0.3

### CODE REVIEW

File: packages/pandoc-abbreviations/pandoc-abbreviations.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+description:
+  "This pandoc filter adds non-breaking spaces after the abbreviations listed in the `abbreviations` file"
+maintainer: ["Samuel Mimram <smimram@gmail.com>"]
+authors: ["Samuel Mimram <smimram@gmail.com>"]
+license: "LGPL-2.1"
+homepage: "https://github.com/smimram/ocaml-pandoc"
+bug-reports: "https://github.com/smimram/ocaml-pandoc/issues"
+depends: [
+  "dune" {>= "2.0"}
+  "pandoc" {>= "0.1.0"}
```
Comment (avsm) [inline packages/pandoc-abbreviations/pandoc-abbreviations.0.1.0/opam:12]:
```suggestion
  "pandoc" {>= "0.1.0"}
  "ocaml" {>="4.04.0"} # due to String.split_on_char
```

**Comment by avsm** [comment]
Very cool package! In your dune-project file, you don't need the dependency on `dune` in the package fields (dune calculates and adds the right one for you) but you do need a dependency on `ocaml` (with the minimum version that you wish to support).

**Comment by smimram** [comment]
Thanks, I have modified the `dune-project` file according to your remarks (for next release)!

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**